app/views/admin/settings/forms/_images.html.erb

```diff
@@ -33,36 +33,7 @@
           <img alt="favicon" class="preview" src="<%= Images::Optimizer.call(Settings::General.favicon_url, width: 32) %>" width="32" height="32" loading="lazy" />
         </div>
 
-        <div class="crayons-field">
-          <%= admin_config_label :logo_png %>
-          <%= admin_config_description Constants::Settings::General::DETAILS[:logo_png][:description] %>
-          <%= f.text_field :logo_png,
-                           class: "crayons-textfield",
-                           value: Settings::General.logo_png,
-                           placeholder: Constants::Settings::General::DETAILS[:logo_png][:placeholder] %>
-          <% if Settings::General.logo_png.present? %>
-            <img src="<%= Images::Optimizer.call(Settings::General.logo_png, width: 128) %>" width="128" height="128" loading="lazy" />
-          <% end %>
-        </div>
```

Comment:
Two quick questions:
1. Am I correct in thinking that we want to remove this `logo_png` field (and potentially the related code to the `logo_png` field) when we release the Creator Onboarding flow since we now have the new and improved logo uploader that allows PNGs and JPEGs?
2. If we do want to remove this `logo_png` field, is it fine to do it in this PR or should I open a follow-up PR so that the focus of this PR remains on removing the Creator Onboarding feature flags?
cc @Ridhwana @nickytonline

Reply:
@juliannatetreault no we don't want to remove `logo_png` at the moment. 

It's completely separate to `logo_svg` is being used in the following places. 

1. The signin modal if you're not logged in but want to comment
2. Some comments view.
3. Onboarding: https://canary1.forem.wtf/onboarding
4. apple-touch-icons
5. sign in invitation when a user is logged out, it shows up on their feed 
6. We expose it via some api calls too.

Perhaps we can do a better job of just explaining the difference with a description for now as it will confuse MANY users. We can tackle this in a separate PR, I'll add it to the board. 

TLDR; 
We will tackle removing logo_png in [this RFC](https://github.com/forem/rfcs/pull/252). At this moment, we will only replace `logo_svg` with `resized_logo`. 

Hope this makes sense :)

Reply:
Thank you so much for the clarification, @Ridhwana! I really appreciate the thorough explanation and will revert this change/merge in [your](https://github.com/forem/forem/pull/15989) image PR once it has been shipped. 😊